### DISCUSSION TITLE
Issue with Pasting Images in Chat Box After Update to Version 4.51

### DISCUSSION THREAD

**Discussion by fabioascoelho** [LiveHelperChat/livehelperchat#2164] [General]
Hi everyone,

Since updating to version 4.51, we’ve encountered an issue where users are unable to paste images (e.g., screenshots) from the clipboard into the chat box using Ctrl+V. This functionality worked perfectly before the update.

Now, when users try to paste an image and press Enter or click the send button, they receive an error message stating that the character limit for sending is 5000 (which is the limit we set in the chat configuration). It seems that the chat is interpreting the pasted image as text.

Has anyone else experienced this issue or have any insights on what might be causing it? Any help would be greatly appreciated!

Thank you!

**Comment by remdex** [comment]
Can you reproduce in official demo?

**Reply by fabioascoelho** [reply]
No, on the official demo, I can send the image with Ctrl+V.
I’m trying to identify what could be wrong with my local installation, since before this update it worked fine. Any suggestions?

**Reply by remdex** [reply]
No, just check that files uploads are active for visitors. Check chrome console for the errors. Also try to clear cache... Without personal look impossible to tell...

**Reply by fabioascoelho** [reply]
Thanks! 
I tried clearing the cache and checking the console, but I couldn’t identify the problem. 
However, I noticed that I was using the Legacy Widget. After switching to the new widget, the ability to paste images with Ctrl+V started working again. Thank you.
